<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-T7322OL" name="Chore tests" tests="1" failures="1" timestamp="26 Jul 2018 07:50:57 GMT" time="74.976" errors="0">
  <testcase name="spicejet" time="19.884" classname="sample.Cross">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at sample.Cross.spicejet(Cross.java:65)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- spicejet -->
</testsuite> <!-- Chore tests -->
